libcnb-cargo/src/main.rs

```diff
@@ -5,6 +5,7 @@
 #![allow(clippy::module_name_repetitions)]
 
 // Suppress warnings due to the `unused_crate_dependencies` lint not handling integration tests well.
+use libcnb_common as _;
```

Comment:
No `#[cfg(test)]` since its an integration test dependency. Adding it would cause Clippy to still complain about an unused dependency.